nav2_behavior_tree/include/nav2_behavior_tree/plugins/condition/is_path_valid_condition.hpp

```diff
@@ -63,7 +63,8 @@ class IsPathValidCondition : public BT::ConditionNode
   {
     return {
       BT::InputPort<nav_msgs::msg::Path>("path", "Path to Check"),
-      BT::InputPort<std::chrono::milliseconds>("server_timeout")
+      BT::InputPort<std::chrono::milliseconds>("server_timeout"),
+      BT::InputPort<uint8_t>("max_cost", 255, "Maximum cost of the path")
```

Comment:
```suggestion
      BT::InputPort<unsigned int>("max_cost", 255, "Maximum cost of the path")
```

Some of the different architectures don't like `uint8_t`, so using base types can be helpful for compiling on a variety of hardware. Please  change this here, in the variable definition in L77, and anywhere else used